### PR TITLE
Added disabled and selected option for select input

### PR DESCRIPTION
**Pull request by severinoAmmirati**
Added disabled and selected option for select input

### CODE REVIEW

**Review by neotrow — CHANGES_REQUESTED** [review]
@severinoAmmirati 

Thanks for the PR 🙏 


@neotob 
As I am on holidays soon, could you take over for the rest of the review? Thanks 🙏

File: src/lib/types/LabelValueOption.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 interface LabelValueOption {
```
[the remark this answers is not in the document]
Comment (severinoAmmirati) [reply src/lib/types/LabelValueOption.ts:1]: @neotob Seems that the disabled prop is already implemented in the Static/AsyncTypeahead. I've added some cypress test to them

File: cypress/cypress/component/Input/Select.cy.tsx [hunk 2]
```diff
@@ -68,3 +68,45 @@ it("is disabled", () => {
...
+it("option is selected", () => {
+  const name = faker.random.word();
+  const options = [...Array<unknown>(5)].map<LabelValueOption>(() => {
+    const randomVal = faker.science.chemicalElement().name;
+    return { label: randomVal, value: randomVal };
+  });
+  options.push({ label: "SelectedOption", value: "SelectedOption", selected: true });
+
+  cy.mount(
+    <Form
```
[the remark this answers is not in the document]
Comment (severinoAmmirati) [reply cypress/cypress/component/Input/Select.cy.tsx]: Test removed, as suggested, selected can be replaced by `defaultValues`

File: src/lib/types/LabelValueOption.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
...
   value: string;
   label: string;
+  disabled?: boolean;
+  selected?: boolean;
```
[the remark this answers is not in the document]
Comment (severinoAmmirati) [reply src/lib/types/LabelValueOption.ts]: Not currently, so it was removed

File: src/lib/types/LabelValueOption.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (neotob) [reply src/lib/types/LabelValueOption.ts:1]: thanks for adding the tests!

**Pull request merged**